metrics/percona/qrt.go

```diff
@@ -39,31 +42,40 @@ const (
 
 const (
 	OPT_PERCENTILES           = "percentiles"
-	OPT_OPTIONAL              = "optional"
+	OPT_REAL_PERCENTILES      = "real-percentiles"
 	OPT_FLUSH_QRT             = "flush"
 	default_percentile_option = "999"
+
+	ERR_UNKNOWN_TABLE = "table-is-unknown"
```

Comment:
Weird... for P_S. events_statements_histogram_global MySQL returns error 1146, but error 1109 for this one. In any case, let's call this `"unknown-table"` to match the const name and MySQL error message.

Reply:
Yeah, the error is different. 
Updated to `unknown-table`